lib/Manifold.js

```diff
@@ -25,18 +25,19 @@
 var _DEBUG = typeof DEBUG === 'undefined' ? false : DEBUG;
 var _ASSERT = typeof ASSERT === 'undefined' ? false : ASSERT;
 
-var common = require('./util/common');
-
-var Vec2 = require('./common/Vec2');
-var Transform = require('./common/Transform');
-var Math = require('./common/Math');
-var Rot = require('./common/Rot');
-
-module.exports = Manifold;
-module.exports.clipSegmentToLine = clipSegmentToLine;
-module.exports.clipVertex = ClipVertex;
-module.exports.getPointStates = getPointStates;
-module.exports.PointState = PointState;
+import common from './util/common';
+
+import Vec2 from './common/Vec2';
+import Transform from './common/Transform';
+import Math from './common/Math';
+import Rot from './common/Rot';
+
+export default Manifold;
+
+Manifold.clipSegmentToLine = clipSegmentToLine;
+Manifold.ClipVertex = ClipVertex;
+Manifold.getPointStates = getPointStates;
+Manifold.PointState = PointState;
```

Comment:
One can also think about exporting these functions via named exports, but I guess this would make it more difficult to correctly export it in `index.js` and maybe it a bit unaesthetic to mix default and named exports. So probably not a good idea 😄 Wanted to at least mention it nevertheless

Reply:
Yup, I did that first, then changed it to keep it backward compatible for now, until we move this out of internal.